python/modules/IcePy/Operation.cpp

```diff
@@ -2249,17 +2249,19 @@ Upcall::dispatchImpl(PyObject* servant, const string& dispatchName, PyObject* ar
         throw Ice::UnknownException{__FILE__, __LINE__, ostr.str()};
     }
 
-    //
-    // Get the _iceDispatch method. The _iceDispatch method will invoke the servant method and pass it the arguments.
-    //
-    PyObjectHandle dispatchMethod{getAttr(servant, "_iceDispatch", false)};
-    if (!dispatchMethod.get())
+    // Get the dispatch function from Ice.PythonDispatcher module.
+    // lookupType() returns a borrowed reference.
+    PyObject* dispatchMethod = lookupType("Ice.PythonDispatcher.dispatch");
```

Comment:
I find the term dispatchMethod confusing. It can be this `dispatch` method or the `method` arg of `dispatch`.

You can't call it simply `dispatch`?